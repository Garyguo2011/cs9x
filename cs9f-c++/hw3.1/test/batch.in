batch test/batch_commands
list names
