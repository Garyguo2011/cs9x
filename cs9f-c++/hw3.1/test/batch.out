apple :  30
orange :  19
orange :  8
banana :  5
apple :  21
apple :  21
banana :  5
orange :  8
banana :  5
orange :  8
apple :  21
